sdk/scheduler/src/main/java/com/mesosphere/sdk/dcos/DcosCertInstaller.java

```diff
@@ -20,7 +20,8 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(DcosCertInstaller.class);
 
     public static final String DEFAULT_JRE_KEYSTORE_PASS = "changeit";
-    public static final String CERT_PATH = ".ssl/ca.crt";
+    public static final String PRE_110_CERT_PATH = ".ssl/ca.crt";
```

Comment:
Can't hurt to add javadocs on these explaining the situation. Also it looks like none of these need to be public?